test/viewmodels/helpers/ItemsAddToListHelperSpec.scala

```diff
@@ -0,0 +1,321 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.helpers
+
+import base.SpecBase
+import controllers.sections.items.routes
+import fixtures.ItemFixtures
+import fixtures.messages.sections.items.ItemsAddToListMessages
+import mocks.services.MockGetCnCodeInformationService
+import models.UnitOfMeasure.{Litres20, Thousands}
+import models.requests.CnCodeInformationItem
+import models.response.referenceData.{BulkPackagingType, CnCodeInformation}
+import models.sections.items.ItemBrandNameModel
+import models.sections.items.ItemBulkPackagingCode.BulkLiquid
+import models.{NormalMode, UserAnswers}
+import org.scalatest.concurrent.IntegrationPatience
+import pages.sections.items._
+import play.api.i18n.Messages
+import play.api.test.FakeRequest
+import play.twirl.api.HtmlFormat
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.{HtmlContent, Text}
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist._
+import uk.gov.hmrc.http.HeaderCarrier
+import viewmodels.checkAnswers.sections.items._
+import views.html.components.{link, span, tag}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class ItemsAddToListHelperSpec extends SpecBase with ItemFixtures with MockGetCnCodeInformationService with IntegrationPatience {
+
+  implicit lazy val ec = app.injector.instanceOf[ExecutionContext]
+  implicit lazy val hc = HeaderCarrier()
+
+  class Setup(userAnswers: UserAnswers = emptyUserAnswers) {
+    implicit lazy val link = app.injector.instanceOf[link]
```

Comment:
Personally, I'd have this injected outside of Setup so multiple `link`, `tag` and `span` aren't in memory. But not a big deal